### PR TITLE
(potential) fix for https://github.com/stackhpc/kayobe/issues/71

### PR DESCRIPTION
**Pull request by autonomouse**

### CODE REVIEW

**Review by markgoddard — COMMENTED** [review]
This is basically the right approach. We don't need kolla_nova_compute_ironic_group to be configurable, so that variable should be removed and hard coded to nova in the inventory.

File: ansible/roles/kolla-ansible/defaults/main.yml [hunk 1]
```diff
@@ -210,6 +210,8 @@ kolla_openstack_logging_debug:
 # Nova options
 #######################
 
+kolla_nova_compute_ironic_host:
```
Comment (markgoddard) [inline ansible/roles/kolla-ansible/defaults/main.yml]: Needs a comment similar to the one below, and the commented out version in etc/kayobe/nova.yml.

File: ansible/roles/kolla-ansible/templates/overcloud-services.j2 [hunk 2]
```diff
@@ -37,11 +37,16 @@ nova
...
-[nova-compute-ironic:children]
 # NOTE: HA for nova-compute services with ironic is still an experimental
 # feature. Provide the option to use a single compute host, even when multiple
 # controllers are in use.
+{% if kolla_nova_compute_ironic_host is not none %}
+[nova-compute-ironic]
+{{ kolla_nova_compute_ironic_host }}
+{% else %}
+[nova-compute-ironic:children]
 {{ kolla_nova_compute_ironic_group }}
```
Comment (markgoddard) [inline ansible/roles/kolla-ansible/templates/overcloud-services.j2]: The default group doesn't need to be configurable - can we set this to just `nova`, then remove all references to kolla_nova_compute_ironic_group?

**Review by markgoddard — CHANGES_REQUESTED** [review]

File: ansible/roles/kolla-ansible/templates/overcloud-services.j2 (hunk 2 above)
Comment (autonomouse) [reply ansible/roles/kolla-ansible/templates/overcloud-services.j2]: done

**Review by markgoddard — APPROVED** [review]
Looks good, thanks

**Pull request merged**